doc/integrating_gps.rst

```diff
@@ -60,16 +60,14 @@ The excellent `nmea_navsat_driver <http://wiki.ros.org/nmea_navsat_driver>`_ pac
     <param name="baud" value="19200"/>
   </node>
 
-This information is only relevant if the user is not manually specifying the origin via the ``datum`` parameter or the ``set_datum`` service.
-
 IMU Data
 ^^^^^^^^
 
 .. note:: Since version 2.2.1, ``navsat_transform_node`` has moved to a standard wherein all heading data is assumed to start with its zero point facing east. If your IMU does not conform to this standard and instead reports zero when facing north, you can still use the ``yaw_offset`` parameter to correct this. In this case, the value for ``yaw_offset`` would be :math:`\pi / 2` (approximately :math:`1.5707963`).
 
 Users should make sure their IMUs conform to `REP-105 <http://www.ros.org/reps/rep-0105.html>`_. In particular, check that the signs of your orientation angles increase in the right direction. In addition, users should look up the `magnetic declination <http://www.ngdc.noaa.gov/geomag-web/#declination>`_ for their robot's operating area, convert it to radians, and then use that value for the ``magnetic_declination_radians`` parameter.
 
-This information is only relevant if the user is not manually specifying the origin via the ``datum`` parameter or the ``set_datum`` service.
+This information is only relevant if the user is not manually specifying the origin via the ``datum`` parameter or the ``set_datum`` service. Note that a ``datum`` is required in order for the ``navsat_transform_node`` to work. This datum will 'null' the GPS sensor by storing the initial pose (latitude, longitude and heading) and using this initial pose for each newly incoming GPS message, of which this initial pose is subtracted in order to create a ``nav_msgs/Odometry``. From this IMU message, only the heading is used; this heading could also be derived from a GPS sensor that outputs a UTM velocity vector, such as the nmea driver in the `nmea_navsat_driver <http://wiki.ros.org/nmea_navsat_driver>`_ package. The published ``TwistStamped`` on the ``fix_velocity`` topic could be used to derive the initial heading by taking the ``atan2`` of the two vector components, if the magnitude of the vector is large enough. This way, no additional IMU is required in order to obtain the initial heading to set the datum of our ``navsat_transform_node``.
```

Comment:
I'm not crazy about this bit, for the following reasons:

1. This section is about IMU data, but this message moves into a general discussion of how `n_t_n` works.
2. I already discuss the datum under "required inputs":
![image](https://cloud.githubusercontent.com/assets/5042191/24351106/6c029f7e-12dc-11e7-80e1-9fb8c13fdb40.png)
Is there  anything I can add to that section that would make it clearer? I think this information belongs there, if you don't think it's already there. I'd also want to adjust the verbiage in that case.
1. The bit about using a GPS to obtain a heading is good to have, but we should add it as a new paragraph, and say something like "An alternative means of generating an earth-referenced heading is..."